telemetry/specs/INT.mdk

```diff
@@ -575,7 +575,7 @@ INT Metadata Header and Metadata Stack:
     0                   1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5     6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   |  Ver  |Rep|C|E|M|R R| Ins Cnt |  Max Hop Cnt  | Total Hop Cnt |
+   |  Ver  |Rep|C|E|M|R R| Ins Cnt |   Reserved    |RemainingHopCnt|
```

Comment:
Wouldn't it be better to make reserved bits contiguous? i.e. Shift Instruction count to the right, adjacent to RemainingHopCnt?

Reply:
Done